spec/services/articles/feed_spec.rb

```diff
@@ -456,7 +447,7 @@
       # We manually called `feed.globally_hot_articles` here because `let` caches it!
       it "still returns articles" do
         empty_feed = false
-        20.times do
+        5.times do
```

Comment:
@joshpuetz you set this to 20 and I am not sure if there was a reason you picked that number but since it is only returning 1 article I decreased it to 5. If there was a reason you choose 20 let me know!

Reply:
I can't recall offhand, but it smells like a hedge against test flakiness. I say ship it!